Services/testService.js

```
@@ -0,0 +1,89 @@
+    }
+  }
+
+  static async getTestResults({ user_id, role }) {
+    try {
+      if (!mongoose.isValidObjectId(test_id)) {
+        throw new Error("testService-get test review: Invalid test id");
+      }
+
+      const query =
+        role === "doctor" ? { doctor: user_id } : { patient: user_id };
+      const test = await this.findTest(test_id);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)